__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET /api/categories", () => {
+  it("should return status 200, responds with array of category objects with properties slug and description", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((response) => {
+        const expectedCategory = {
+          slug: expect.any(String),
+          description: expect.any(String),
+        };
+        const categoriesArray = response.body.categories;
+        categoriesArray.forEach((category) => {
```

Comment:
when we're looping through the response it's essential to check the length of the array is what you expect. If the response was an empty array here the test would give you a false positive